content/modules/ROOT/pages/module-02.adoc

```diff
@@ -157,6 +157,10 @@ image::02/02-create-workbench.png[]
 ** **Accelerator**: None
 ** **Persistent storage size**: 5 GiB (default)
 
++
+[.bordershadow]
+image::02/02-create-workbench-full.png[]
```

Comment:
I suggest opening this in a new tab, it is very small to see.

Reply:
I split in two images so they are a bit larger.